lib/Transforms/InsertGPUAllocs.cpp

```diff
@@ -545,7 +551,7 @@ class InsertGPUAllocsPass final
     // with gpu.alloc and insert a memref.copy from host to device
     for (const auto &it : gpuBufferParams) {
       auto param = block.getArgument(it.first);
-      if (isGpuAddrSpace(param))
+      if (isGpuAddrSpace(param) || isUsmArgs.getValue())
```

Comment:
I think would be better to put the entire loop into `if (!isUsmArgs.getValue()) { for (...`.

Reply:
Thanks for the advice! I move the `if` out of the loop.(56cf8f1)